File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int mul = 1;
        int com = 0;
        foreach(var item in num_list)
        {
            mul *= item;
            com += item;
        }
        answer = mul < com * com ? 1 : 0; 
        
        return answer;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    int mul = 1;
    int com = 0;
    for(auto num : num_list)
    {
        mul *= num;
        com += num;
    }
    answer = mul < com * com ? 1 : 0; 
    return answer;
}